<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>728</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="1,3,4,1,1,1">
    <item>
     <widget class="QPushButton" name="btnCompile">
      <property name="text">
       <string>Compile</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="Altimeter_Widget" name="Altimeter_display" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>3</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Horizon_Widget" name="Horizon_display" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>3</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Heading_Widget" name="Heading_display" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>3</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Orientation_Widget" name="Orientation_display" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>5</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="leftMargin">
        <number>2</number>
       </property>
       <property name="topMargin">
        <number>2</number>
       </property>
       <property name="rightMargin">
        <number>2</number>
       </property>
       <property name="bottomMargin">
        <number>2</number>
       </property>
       <item>
        <widget class="Plot2d_Widget" name="xy_mag" native="true"/>
       </item>
       <item>
        <widget class="Plot2d_Widget" name="xz_mag" native="true"/>
       </item>
       <item>
        <widget class="Plot2d_Widget" name="yz_mag" native="true"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="ValueBar_Widget" name="vb_mx" native="true"/>
       </item>
       <item>
        <widget class="ValueBar_Widget" name="vb_my" native="true"/>
       </item>
       <item>
        <widget class="ValueBar_Widget" name="vb_mz" native="true"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lblCycles">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lblCenter">
      <property name="text">
       <string>Center:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>728</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Heading_Widget</class>
   <extends>QWidget</extends>
   <header>widgets/heading_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Horizon_Widget</class>
   <extends>QWidget</extends>
   <header>widgets/horizon_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Altimeter_Widget</class>
   <extends>QWidget</extends>
   <header>widgets/altimeter_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Orientation_Widget</class>
   <extends>QWidget</extends>
   <header>widgets/orientation_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Plot2d_Widget</class>
   <extends>QWidget</extends>
   <header>widgets/plot2d_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ValueBar_Widget</class>
   <extends>QWidget</extends>
   <header>widgets/valuebar_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
